### PR TITLE
Claims

### PR DESCRIPTION
**Pull request by Iamsamwhoami**

### CODE REVIEW

File: Src/Screens/ClaimsScreen/Claims.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 15
+        
+        let claimsHeader = ClaimsHeader(presentingViewController: viewController)
+        bag += stackView.addArangedSubview(claimsHeader)
+        
+        let commonClaimsCollection = CommonClaimsCollection(presentingViewController: viewController)
+
```
Comment (axelbacklund) [inline Src/Screens/ClaimsScreen/Claims.swift]: NIT: remove empty new line

File: Src/Screens/ClaimsScreen/Viewables/CommonClaimsCollection/CommonClaimsCollection.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                height: 140
+            )
+        }
+        
+        bag += collectionKit.delegate.willDisplayCell.onValue({ cell, indexPath in
+            cell.layer.zPosition = CGFloat(indexPath.row)
+        })
+        
+        bag += client.fetch(query: CommonClaimsQuery(locale: Localization.Locale.currentLocale.asGraphQLLocale())).onValue { result in
+            let rows = result.data!.commonClaims.enumerated().map {
```
Comment (axelbacklund) [inline Src/Screens/ClaimsScreen/Viewables/CommonClaimsCollection/CommonClaimsCollection.swift]: Maybe dangerous to force unwrap data?

File: Src/Screens/ClaimsScreen/Claims.swift (hunk 1 above)
Comment (Iamsamwhoami) [reply Src/Screens/ClaimsScreen/Claims.swift]: Swiftformat to the rescue 😍

File: Src/Screens/ClaimsScreen/Viewables/CommonClaimsCollection/CommonClaimCard.swift [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+        contentView.addSubview(expandedHeaderView)
+        
+        expandedHeaderView.snp.makeConstraints { make in
+            make.height.equalTo(100)
+            make.top.equalTo(0)
+            make.left.equalTo(0)
+            make.width.equalToSuperview()
+        }
+        
+        let titleLabel = UILabel(value: data.title, style: .rowTitle)
```
Comment (axelbacklund) [inline Src/Screens/ClaimsScreen/Viewables/CommonClaimsCollection/CommonClaimCard.swift]: Maybe MultilineLabel to accommodate for longer strings?

File: Src/Screens/ClaimsScreen/Screens/Viewables/BulletPointCard.swift [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        cardContainer.layer.shadowColor = UIColor.black.cgColor
+        cardContainer.layer.shadowOpacity = 0.05
+        
+        view.addArrangedSubview(cardContainer)
+        
+        let contentView = UIStackView()
+        contentView.axis = .vertical
+        contentView.alignment = .top
+        contentView.spacing = 5
+        contentView.layoutMargins = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
```
Comment (axelbacklund) [inline Src/Screens/ClaimsScreen/Screens/Viewables/BulletPointCard.swift]: Standardize margins as discussed

**Pull request merged**